install/mysql/glpi-empty.sql

```diff
@@ -686,6 +686,7 @@ CREATE TABLE `glpi_changetasks` (
   `begin` datetime DEFAULT NULL,
   `end` datetime DEFAULT NULL,
   `users_id` int(11) NOT NULL DEFAULT '0',
+  `editors_id` int(11) NOT NULL DEFAULT '0',
```

Comment:
it's better to call the field users_id_editor, as it's done for example for users_id_tech